stonesoup/sensormanager/base.py

```diff
@@ -23,8 +27,11 @@ class SensorManager(Base, ABC):
     which communicate with other sensor managers in a networked fashion.
 
     """
-    sensors: Set[Sensor] = Property(doc="The sensor(s) which the sensor manager is managing. "
-                                        "These must be capable of returning available actions.")
+    sensors: Set['Sensor'] = Property(
+        default=set(), doc="The sensor(s) which the sensor manager is managing.")
+
+    platforms: Set['Platform'] = Property(
+        default=set(), doc="The platform(s) which the sensor manager is managing.")
```

Comment:
```suggestion
    sensors: Set['Sensor'] = Property(
        default=None, doc="The sensor(s) which the sensor manager is managing.")

    platforms: Set['Platform'] = Property(
        default=None, doc="The platform(s) which the sensor manager is managing.")
```

These need to be default=None and then can create empty sets in the init function

Reply:
I've approved PR, but based on assumption this will be fixed 👍